product_brand/views/product_brand_view.xml

```diff
@@ -40,7 +40,7 @@
                                 name="products_count"
                                 widget="statinfo"
                                 string="Products"
-                                context="{'default_product_brand_id': active_id}"
+                                context="{'default_product_brand_id': id}"
```

Comment:
this context is useless, should move it to < button/> instead:
```
<button
                            name="%(action_open_brand_products)d"
                            type="action"
                            class="oe_stat_button"
                            icon="fa-cubes"
context="{'default_product_brand_id': id}">
```

Reply:
Updated, thank @nguyenminhchien